downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```diff
@@ -1,33 +1,44 @@
+
 :imagesdir: images
 = Managing Red Hat Ansible Content Collections and Ansible Galaxy collections in automation hub
 :numbered:
 :experimental:
 
 include::attributes/attributes.adoc[]
 
-You can sync {HubName} to use the Red Hat Certified Collections available through your {PlatformNameShort} subscription or community collections available through {Galaxy}.
+Included with a subscription to {PlatformName}, {CertifiedColl} feature {CertifiedName} and {Valid}. 
+Using {HubNameMain}, your organization can access and curate a unique set of collections from all forms of Ansible content. 
+
+You can also synchronize {HubNameMain} to create a custom list of collections from either {CertifiedCon}, {valid}, or {Galaxy} community content. 
 
-Your organization can access and curate into a unique set of collections from all Red Hat Certified content in the {HubName} service hosted on console.redhat.com.
+At present, Ansible validated collections are only available in your {PrivateHubName} {Valid} as part of the Platform Installer.
+When you download the bundled installer by default Validated Content is pre-populated into the {PrivateHubName}, if the {PrivateHubName} is enabled as part of the inventory.
 
-You can also configure private {HubName} for signing and publishing Ansible content collections tailored to meet your organization's unique needs.
+These collections can be updated manually by downloading their packages.
 
 include::aap-common/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 include::hub/assembly-synclists.adoc[leveloffset=+1]
 
 include::hub/assembly-syncing-to-cloud-repo.adoc[leveloffset=+1]
 
+include::hub/assembly-private-automation-hub.adoc[leveloffset=+1]
+
 include::hub/assembly-collections-and-content-signing-in-pah.adoc[leveloffset=+1]
 
+include::hub/assembly-faq.adoc[leveloffset=+1]
+
+include::hub/assembly-validated-content.adoc[leveloffset=+1]
+
 == Conclusion
 
 When you complete all of the previous procedures, you will have:
```

Comment:
I don't see conclusions very often. I wonder if this list should come at the beginning in the abstract/short description before the first procedure.